tests/network/vmi_multus.go

```diff
@@ -331,26 +331,41 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
+			getFedoraVmi := func(interfaces []v1.Interface, networks []v1.Network, cloudInitNetworkData string) *v1.VirtualMachineInstance {
+				withVmiOptions := []libvmi.Option{
+					libvmi.WithCloudInitNoCloudNetworkData(cloudInitNetworkData, false),
+				}
+				for idx, _ := range networks {
```

Comment:
I'm quite sure you can drop the `, _` part here; you can use just an index when iterating a slice.

Reply:
Fixed